packages/reactive-element/src/reactive-element.ts

```diff
@@ -728,21 +729,22 @@ export abstract class ReactiveElement
     key: string | symbol,
     options: PropertyDeclaration
   ): PropertyDescriptor | undefined {
+    const {get, set} = getOwnPropertyDescriptor(this.prototype, name) ?? {
```

Comment:
Ah, I got surprised by https://google.github.io/styleguide/tsguide.html#static-this . No action required. Is it worth doing something like `ReactiveElement.prototype`. I guess all over the codebase we already use a static `this`.

Reply:
It can't be `ReactiveElement.prototype` because we're calling into a potentially overridden method.

Reply:
Peter just gave me a great rundown on how all of this works. Yep I understand.